packages/open-next/src/build.ts

```diff
@@ -685,6 +685,9 @@ async function createCacheAssets(monorepoRoot: string) {
 export function compileCache(format: "cjs" | "esm" = "cjs") {
   const ext = format === "cjs" ? "cjs" : "mjs";
   const outfile = path.join(options.outputDir, ".build", `cache.${ext}`);
+
+  const isAfter15 = compareSemver(options.nextVersion, "14.9.9") >= 0;
```

Comment:
Probably unlikely that vercel will do a 14.9 and up, but wouldn't `compareSemver(options.nextVersion, "15.0.0") >= 0` be more clear?

Reply:
Ho yeah right, i was testing things here and i thought that it was this that was causing trouble. It was not.
I'll change to `15.0.0`